### ISSUE TITLE
Bug Overriden classes

### ISSUE THREAD

**Issue by MarioCadenas** [negomi/react-burger-menu#251]
Hi, if you try to pass a className to one of the items in the menu it gets overrided by the new logic of the bmItem in v2.5.0

example:
```
class MyComponent extends React.Component {
  render(){
    return (
      <Menu styles={this.menuStyles} itemClassName="my-custom-item">
        <Link to="/a">A</Link>
        <Link className="class-only-for-this-item" to="/b">B</Link>
      </Menu>
    )
  }
}
```

In this case, both "Link" tags will get class "bm-item" and "my-custom-item", but second "Link" tag will get his own class removed. Hope i explained well, thanks!

**Comment by negomi** [comment]
Thanks @MarioCadenas, I'll look into it!

**Comment by negomi** [comment]
Fixed in 2.5.1 :)
